posthog/api/event_definition.py

```diff
@@ -54,28 +68,54 @@ def get_queryset(self):
             else:
                 search = self.request.GET.get("search", None)
                 search_query, search_kwargs = term_search_filter_sql(self.search_fields, search)
+
+                params = {
+                    "team_id": self.team_id,
+                    "included_ids": included_event_ids,
+                    "excluded_ids": excluded_event_ids,
+                    **search_kwargs,
+                }
+
                 # Prevent fetching deprecated `tags` field. Tags are separately fetched in TaggedItemSerializerMixin
                 event_definition_fields = ", ".join(
                     [f'"{f.column}"' for f in EnterpriseEventDefinition._meta.get_fields() if hasattr(f, "column") and f.column != "tags"]  # type: ignore
                 )
 
                 ee_event_definitions = EnterpriseEventDefinition.objects.raw(
                     f"""
-                    SELECT {event_definition_fields}
+                    SELECT {event_definition_fields},
+                           {included_event_ids_field} AS is_included_event
                     FROM ee_enterpriseeventdefinition
                     FULL OUTER JOIN posthog_eventdefinition ON posthog_eventdefinition.id=ee_enterpriseeventdefinition.eventdefinition_ptr_id
-                    WHERE team_id = %(team_id)s {search_query}
-                    ORDER BY query_usage_30_day DESC NULLS LAST, last_seen_at DESC NULLS LAST, name ASC
+                    WHERE team_id = %(team_id)s AND (
+                        {included_event_ids_field} = true
```

Comment:
Not sure if this borks something, but should be fine?

```suggestion
                        is_included_event = true
```

Reply:
I tried this out too but turns out you can't use new column names in the WHERE clause without throwing a syntax error.